packages/core/upload/admin/src/components/AssetCard/tests/ImageAssetCard.test.js.orig

```diff
@@ -0,0 +1,702 @@
+import React from 'react';
```

Comment:
Is this file needed?

Reply:
No. You are right. It's a leftover from a bad merge. Good catch, thank you!

Removed in 48e60db5c36b7bc0225a39363928021ab25bccee